sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/misc.scala

```diff
@@ -177,3 +177,20 @@ case class Version() extends LeafExpression with CodegenFallback {
     UTF8String.fromString(SPARK_VERSION_SHORT + " " + SPARK_REVISION)
   }
 }
+
+@ExpressionDescription(
+  usage = """_FUNC_(expr) - Return readable string representation for the type.""",
```

Comment:
`for the type` -> `for the data type of the input`

Reply:
thanks.